ci-operator/step-registry/upi/gcp/post/upi-gcp-post-chain.yaml

```diff
@@ -0,0 +1,7 @@
+chain:
+  as: upi-gcp-post
+  steps:
```

Comment:
i think this is missing the `gather` chain like  https://github.com/openshift/release/blob/f6b3a1c5360a4f67f24a832986f5ffaea357f979/ci-operator/step-registry/ipi/deprovision/ipi-deprovision-chain.yaml#L4

Reply:
Nope... it calls ipi-deprovision-chain, which includes gather.